### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/24.09/ai_%E4%B8%AA%E4%BA%BA%E7%94%A8%E6%88%B7%E7%AE%80%E5%8D%95%E5%85%8D%E8%B4%B9%E6%8E%A5%E5%85%A5gpt%E7%9A%84%E6%96%B9%E5%BC%8F

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#62] [General]
# https://www.xiaogenban1993.com/blog/24.09/ai_%E4%B8%AA%E4%BA%BA%E7%94%A8%E6%88%B7%E7%AE%80%E5%8D%95%E5%85%8D%E8%B4%B9%E6%8E%A5%E5%85%A5gpt%E7%9A%84%E6%96%B9%E5%BC%8F

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/24.09/ai_%E4%B8%AA%E4%BA%BA%E7%94%A8%E6%88%B7%E7%AE%80%E5%8D%95%E5%85%8D%E8%B4%B9%E6%8E%A5%E5%85%A5gpt%E7%9A%84%E6%96%B9%E5%BC%8F

**Comment by sunwu51** [comment]
目前遇到的限制是，github models的gpt-4o模型上下文不能超过8K。
